src/main/java/tds/testpackageconverter/converter/mappers/TestPackageBlueprintMapper.java

```diff
@@ -269,6 +299,7 @@ private static BlueprintElement mapAssessmentBlueprintElements(final Testspecifi
                                 .filter(bpEl -> bpEl.getElementtype().equalsIgnoreCase(BlueprintElementTypes.STRAND))
                                 .map(bpEl -> BlueprintElement.builder()
                                         .setId(bpEl.getIdentifier().getName())
+                                        .setLabel(csidEnhancer.enhance(bpEl.getIdentifier().getName()))
```

Comment:
What would you think about removing state from the `CsidEnhancer` class?  It doesn't seem like the class needs state and the `enhance` method could be a single static call in the class that does the conversion taking in the subject and grade.

Reply:
It's certainly doable, but I just feel it's a little cleaner this way. Again, I'll change it if you feel strongly about it.